recipes/rl-games/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "rl_games" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Denys88/rl_games/archive/v{{ version }}.tar.gz
+  sha256: f70429cb4c75972365362186dc7b33d90b67e543fc770b272bb812b914f44e5a
+  patches:
+    - 01-exclude_package_data.patch
+    - 02-avoid_poetry_solver.patch
+    - 03-remove_watchdog_upper_bound.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >3.7
+    - pip
+  run:
+    - python >3.7
+    - gym >=0.17.2
+    - tensorboard >=1.14.0
+    - tensorboardx >=1.6
+    - pyyaml
+    - psutil
+    - setproctitle
+    - opencv
+    - wandb
+    - pytorch
+    - numpy
+    - watchdog >=2.19
+  run_constrained:
+    - ale-py
+    - AutoROM
+    - brax
+    - jax
+    - mujoco-py
+    - envpool
```

Comment:
This is a bit strange. `run_constrained` is used to enforce constraint on packages that are not run dependencies, if there are no constraints why you want to have them as `run_constrained` ?

Reply:
I think it's a leftover from the first version of the recipe that took into account the constraints from the original `setup.py`. Thanks!